### ISSUE TITLE
Hello, when I reproduced this paper, I found that the Umamba.py under modelszoo was missing, can you send it to me?

### ISSUE THREAD

**Issue by Nooo828** [zs1314/OCTAMamba#4]
Or did you get this file by some other means?

**Comment by zs1314** [comment]
@Nooo828  Yes, I reproduced it using the official source code. Due to the high encapsulation of their code, I couldn't easily integrate it into my own code framework, so I directly used their code framework to obtain the experimental data for U-Mamba. I can provide you with the link: https://github.com/bowang-lab/U-Mamba.

**Comment by zs1314** [comment]
I think this is difficult because they use the nn-Unet project architecture, making it hard for me to extract it. The best approach would be for you to run their project yourself. @Nooo828
